### PR TITLE
Promote staging to prod

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

**Comment by Br3nda** [comment]
Probably should include #80 too

File: .rubocop.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@
 # https://github.com/bbatsov/rubocop/blob/master/config/default.yml
 
 # We do not typically use/need class documentation
+Lint/Void:
```
Comment (Br3nda) [inline .rubocop.yml]: This no longer matches with the comment above

**Comment by Br3nda** [comment]
"This branch cannot be rebased due to conflicts" 
Did someone push directly to master?

**Comment by elspeth-rabid** [comment]
I see not evidence of that in the commit history.

**Pull request merged**